### PR TITLE
[lithium] bump deployment target to iOS 10.3

### PR DESCRIPTION
**Pull request by jayahariv**

### CODE REVIEW

**Review by pasin — COMMENTED** [review]
10.3 could be right but I don't remember why it is not 10.0?

**Comment by jayahariv** [comment]
Since 10.0 and 10.3 are similar but with some extra features without any API breaking changes. Which makes sense for any users to adopt it seamlessly.

**Comment by pasin** [comment]
If I have an app that set the target deployment to 10.0, can the app's project use CBL with 10.3 as the target deployment?

**Comment by pasin** [comment]
IMO, Setting it to 10.0 doesn't require to verify and doesn't cause any problems.

**Comment by jayahariv** [comment]
> If I have an app that set the target deployment to 10.0, can the app's project use CBL with 10.3 as the target deployment?
No. they need to upgrade that to 10.3.

File: xcconfigs/Project.xcconfig [hunk 1]
```diff
@@ -22,9 +22,9 @@ CBL_BUILD_NUMBER                   = 0
 CBL_EXPORTED_SYMBOLS_FILE          = Objective-C/CouchbaseLite.exp
 CBL_SWIFT_MODULEMAP_FILE           = Swift/CouchbaseLiteSwift.modulemap
 
-IPHONEOS_DEPLOYMENT_TARGET         = 9.0
+IPHONEOS_DEPLOYMENT_TARGET         = 10.3
```
Comment (jayahariv) [inline xcconfigs/Project.xcconfig:25]:
```suggestion
IPHONEOS_DEPLOYMENT_TARGET         = 10.0
```

File: xcconfigs/Project.xcconfig [hunk 2]
```diff
@@ -22,9 +22,9 @@ CBL_BUILD_NUMBER                   = 0
...
 MACOSX_DEPLOYMENT_TARGET           = 10.12
-TVOS_DEPLOYMENT_TARGET             = 9.0
+TVOS_DEPLOYMENT_TARGET             = 10.3
```
Comment (jayahariv) [inline xcconfigs/Project.xcconfig:27]:
```suggestion
TVOS_DEPLOYMENT_TARGET             = 10.0
```

**Review by pasin — APPROVED** [review]

**Pull request merged**